### ISSUE TITLE
[Pagetual] Bug report: IP Get banned from some websites caused by using Pagetual

### ISSUE THREAD

**Issue by tonyycdd** [hoothin/UserScripts#691]
**Describe the bug**
As written in the title. Just click in the website and scoll a bit and refresh the page,  then my IP will git banned.
Some examples: websites using Cloudflare, and also ehentai. 

I know that we have blacklist feature for Pagetual, but I wish to have a whitelist mode or something so that I wouldn't get accidently banned before I can browse the websites.

**Desktop (please complete the following information):**
 - OS: Windows10
 - Browser [chrome]

**Comment by hoothin** [comment]
Yes, there is a whitelist mode, you can just uncheck that option. However, I would like to note that Pagetual fetches the next page using a simple GET request, so the response itself is unlikely to be the reason you get banned. I'm also a fan of exhentai and have not encountered any issues there.
![image](https://github.com/user-attachments/assets/438588ca-36f9-49ee-881c-022d40814dd9)
